en/docs/learn/api-gateway/message-mediation/changing-the-default-mediation-flow-of-api-requests.md

```diff
@@ -147,6 +148,9 @@ You can copy this content into an XML file (e.g., `global_ext.xml` ) and save it
 !!! tip
     If you are having a distributed setup, do the changes in **Gateway** node.
 
+!!! note
+    The mediation extension is applied to all resources of the API. Therefore, all resources will contain a similar logic to construct the endpoint.
```

Comment:
@ruthryi  This document contains details about mediation sequences in general. Let's remove the "Therefore, all resources will contain a similar logic to construct the endpoint." part here.